src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -60,7 +62,7 @@ const WorkspaceTabs = ({
       tabNames: _.map('name', tabs),
       getHref: currentTab => Nav.getLink(_.find({ name: currentTab }, tabs).link, { namespace, name })
     }, [
-      isAnalysisTabVisible() ? h(Fragment) : h(WorkspaceMenuTrigger, { canShare, isOwner, setCloningWorkspace, setSharingWorkspace, setDeletingWorkspace }, [
+      isAnalysisTabVisible() ? h(Fragment) : h(WorkspaceMenuTrigger, { canShare, isLocked, namespace, name, isOwner, setCloningWorkspace, setSharingWorkspace, setTogglingWorkspaceLock, setDeletingWorkspace }, [
```

Comment:
nit: formatting only
```suggestion
      isAnalysisTabVisible() ? h(Fragment) : h(WorkspaceMenuTrigger, {
        canShare, isLocked, namespace, name, isOwner, setCloningWorkspace, setSharingWorkspace,
        setTogglingWorkspaceLock, setDeletingWorkspace
      }, [
```